domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/displaygroup/TabShowConditionValidatorImpl.java

```diff
@@ -43,7 +46,7 @@ public ValidationResult validate(DisplayGroupEntity thisDisplayGroup, List<Displ
                 return validationResult;
             }
 
-            List<String> allSubTypePossibilities = CaseFieldEntityUtil
+            List<String> allSubTypePossibilities = caseFieldEntityUtil
```

Comment:
this block of code is very similar if not identical to the one below and the one in PageShowConditionValidatorImpl. We are introducing some duplication. I think we should extract this into its own class, called something like FieldsPathValidator or sim